pkg/multicluster/link.go

```diff
@@ -121,6 +123,18 @@ func NewLink(u unstructured.Unstructured) (Link, error) {
 		return Link{}, err
 	}
 
+	selector := metav1.LabelSelector{}
```

Comment:
Maybe worth putting a comment that this piece of code is to exercise the serde rountrip